models/database.go

```diff
@@ -597,6 +597,17 @@ var databaseSchema = [][]string{
 			ADD COLUMN timeout BIGINT
 		`,
 	},
+	46: {
+		`CREATE TABLE job_logs (
+			job_id VARCHAR NOT NULL,
+			chunk_id INTEGER NOT NULL,
+			message TEXT NOT NULL,
+			last_chunk BOOLEAN NOT NULL,
+			created_at TIMESTAMP NOT NULL,
+			FOREIGN KEY (job_id) REFERENCES jobs (id) ON DELETE CASCADE,
+			UNIQUE (job_id, chunk_id)
```

Comment:
Better to have `PRIMARY KEY` here as we this data will be always grabbed by these fields combination.

Reply:
REFORM doesn't support PRIMARY KEY with multiple columns.

Reply:
ORM stuff 😔

Reply:
I changed it to primary key. But we have to construct query ourselves.
https://github.com/go-reform/reform/issues/114